mreg/api/v1/tests/test_networks.py

```diff
@@ -525,6 +525,9 @@ def test_reject_ip_usage(self):
     def test_count_unused_ips(self):
         """Test that excluded ipaddresses are exempted from the unused count"""
         self.test_create()
+        # Create a host that uses one of the reserved addresses. (Should not be counted twice)
+        host1 = Host.objects.create(name='host1.example.org')
+        Ipaddress.objects.create(host=host1, ipaddress='10.0.0.255')
```

Comment:
Technically not one of the reserved address, as 10.0.0.255 is the broadcast address. The reserved addresses are the N (usually 3) first addresses, i.e. 10.0.0.{1,2,3}.

Could add another `Ipaddress` with e.g. 10.0.0.2 just to make sure it is counted properly.